packages/core/docs/commercetools/enterprise/use-review.md

```diff
@@ -2,40 +2,21 @@
 platform: Commercetools
 ---
 
-<IncludeContent content-key="use-review" />
+# Reviews
 
-::: slot search-params
+[[toc]]
 
-```typescript
-interface ReviewSearchParams {
-  productId: string;
-  limit?: number;
-  offset?: number;
-}
-```
+## Features
 
-:::
+<IncludeContent content-key="useReview/features" />
```

Comment:
I'm not sure if thats the right approach, we usually have a different sets of features for enterprise packages (putted in a context of a specific integration). If it fits now its fine but please dont treat that as a source for truth in the future and dont try to keep it agnostic ;)